cabal.project.release

```diff
@@ -1,11 +1,23 @@
-packages: ./ghcup.cabal
-
-optional-packages: ./vendored/*/*.cabal
+import: cabal.project
 
 optimization: 2
 
-package ghcup
-  flags: +tui -tar
+if impl(ghc < 9.0)
+    package ghcup
+        flags: +tui -tar
+else
+    package ghcup
+        flags: +tui +tar
+
+constraints: http-io-streams -brotli,
```

Comment:
Why add these when you already did `import: cabal.project`?

Reply:
True, though by that logic, shouldn't `cabal.project.release` be simplified to:
```cabal
import: cabal.project

optimization: 2

if os(linux)
  if arch(x86_64) || arch(i386)
    package *
      ghc-options: -split-sections -optl-static
elif os(darwin)
  constraints: zlib +bundled-c-zlib,
               lzma +static
elif os(freebsd)
  constraints: zlib +bundled-c-zlib,
               zip +disable-zstd
  package *
    ghc-options: -split-sections
```

Reply:
BTW, these constraints already existed at the bottom of the file -- I just moved them to the same place they were in the other `cabal.project` files